backend/compact-connect/lambdas/provider-data-v1/tests/function/test_handlers/test_bulk_upload.py

```diff
@@ -0,0 +1,88 @@
+import json
+from uuid import uuid4
+
+from botocore.exceptions import ClientError
+from moto import mock_aws
+from tests.function import TstFunction
+
+
+@mock_aws
+class TestBulkUpload(TstFunction):
+    def test_get_bulk_upload_url(self):
+        from handlers.bulk_upload import bulk_upload_url_handler
+
+        with open('tests/resources/api-event.json', 'r') as f:
+            event = json.load(f)
+
+        event['requestContext']['authorizer']['claims']['scope'] = 'openid email stuff aslp/oh.write'
+        event['pathParameters'] = {
+            'compact': 'aslp',
+            'jurisdiction': 'oh'
+        }
+        resp = bulk_upload_url_handler(event, self.mock_context)
+
+        self.assertEqual(200, resp['statusCode'])
+        body = json.loads(resp['body'])
+        self.assertEqual({'url', 'fields'}, body['upload'].keys())
+
+    def test_get_bulk_upload_url_forbidden(self):
+        from handlers.bulk_upload import bulk_upload_url_handler
+
+        with open('tests/resources/api-event.json', 'r') as f:
+            event = json.load(f)
+        event['requestContext']['authorizer']['claims']['scope'] = 'openid email stuff aslp/ne.write'
+        event['pathParameters'] = {
+            'compact': 'aslp',
+            'jurisdiction': 'oh'
+        }
+
+        resp = bulk_upload_url_handler(event, self.mock_context)
+
+        self.assertEqual(403, resp['statusCode'])
+
+    def test_get_no_auth_bulk_upload_url(self):
+        from handlers.bulk_upload import no_auth_bulk_upload_url_handler
+
+        with open('tests/resources/api-event.json', 'r') as f:
+            event = json.load(f)
+        del event['requestContext']['authorizer']
+        event['pathParameters'] = {
+            'compact': 'aslp',
+            'jurisdiction': 'oh'
+        }
+
+        resp = no_auth_bulk_upload_url_handler(event, self.mock_context)
```

Comment:
Oh, I should remove this method - it's dead code at this point. Short answer, nothing happens because there is no endpoint mapped to this handler anymore.